libs/ui/tabs/brain/src/lib/brn-tabs-trigger.directive.ts

```diff
@@ -8,24 +8,26 @@ import { BrnTabsDirective } from './brn-tabs.directive';
 		'[id]': 'labelId',
 		type: 'button',
 		role: 'tab',
-		'[tabindex]': '_isSelected() ? "0": "-1"',
-		'[attr.aria-selected]': '_isSelected()',
+		'[tabindex]': 'selected() ? "0": "-1"',
+		'[attr.aria-selected]': 'selected()',
 		'[attr.aria-controls]': 'contentId',
-		'[attr.data-state]': "_isSelected() ? 'active' : 'inactive'",
+		'[attr.data-state]': "selected() ? 'active' : 'inactive'",
 		'[attr.data-orientation]': '_orientation()',
 		'[attr.data-disabled]': "disabled ? '' : undefined",
 		'(click)': 'activate()',
```

Comment:
We should probably start adding the `exportAs` for all of the brains 🧠 Thoughts?